<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DruidCentralWidget</class>
 <widget class="QWidget" name="DruidCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Mashing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Boiling</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Fermentation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Time</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1" rowspan="5">
         <widget class="QLCDNumber" name="_time_lcd"/>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Hour</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="_time_hour"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Minutes</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="_time_minutes"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="_time_start">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="_time_stop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Temperature</string>
       </property>
       <zorder>groupBox_6</zorder>
       <zorder>groupBox_6</zorder>
       <zorder>groupBox_6</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Volume</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox_3</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
